packages/blade/src/components/Chip/__tests__/ChipGroup.web.test.tsx

```diff
@@ -0,0 +1,268 @@
+import userEvents from '@testing-library/user-event';
+import React from 'react';
+import { Chip } from '../Chip';
+import type { ChipGroupProps } from '../ChipGroup/ChipGroup';
+import { ChipGroup } from '../ChipGroup/ChipGroup';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+
+const selectionTypes: ChipGroupProps['selectionType'][] = ['single', 'multiple'];
+const sizes: ChipGroupProps['size'][] = ['xsmall', 'small', 'medium', 'large'];
+const intents: ChipGroupProps['intent'][] = ['neutral', 'positive', 'negative'];
+
+describe('<ChipGroup />', () => {
+  for (const selectionType of selectionTypes) {
+    describe(`selectionType="${selectionType}"`, () => {
+      it(`should render with selectionType="${selectionType}"`, () => {
+        const { container, getByRole } = renderWithTheme(
+          <ChipGroup accessibilityLabel="Select fruits" selectionType={selectionType}>
+            <Chip value="apple">Apple</Chip>
+            <Chip value="mango">Mango</Chip>
+            <Chip value="orange">Orange</Chip>
+          </ChipGroup>,
+        );
+
+        const chipRole = selectionType === 'single' ? 'radio' : 'checkbox';
+        expect(getByRole('group')).toBeInTheDocument();
```

Comment:
will the roll be a group for a radio as well?

Reply:
@anuraghazra Can you confirm? Currently, I see we have implemented `group` in radio web and `radiogroup ` on RN
https://github.com/razorpay/blade/blob/f13a18d51cfd1b013fa7d573a724d4dba4eee000/packages/blade/src/components/Radio/RadioGroup/RadioGroup.tsx#L139

Reply:
Actually in web & native both should be `radiogroup` only, i'm not sure why i put `group` on web for RadioGroup seems like a mistake.

Reply:
I'll update it for Chip. For radio I'll open a separate PR.